.evergreen/config_generator/etc/utils.py

```diff
@@ -23,11 +23,23 @@ class Task(EvgTask):
 
 
 # Automatically formats the provided script and invokes it in Bash.
-def bash_exec(script, **kwargs):
+def bash_exec(
+    script,
+    *,
+    include_expansions_in_env: Iterable[str] | None = None,
+    working_dir: str | None = None,
+    command_type: EvgCommandType | None = None,
+    env: Mapping[str, str] | None = None,
```

Comment:
Is this really necessary? The point of the `bash_exec` helper is to be a lightweight abstraction for the common `bash -c '<script'>` pattern; all other parameters are directly forwarded to `subprocess_exec` for it to parse and validate. I would like to avoid having to maintain compatibility/consistency of the parameter list with that of `subprocess_exec`.

Reply:
The purpose of explicitly enumerating the parameters we want is to early-detect breaking changes and enable tool assistance. If Shrub changes/removes parameters in a way that causes a mismatch, I want tooling to call out the incompatibility. Using a bare `**kwargs` is opaque to tools.